vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_monthly.cc

```diff
@@ -89,13 +89,15 @@ void ContributionMonthly::GetVerifiedTipList(
     non_verified.push_back(std::move(contribution));
   }
 
-  if (!non_verified.empty()) {
-    auto save_callback = std::bind(
-        &ContributionMonthly::OnSavePendingContribution,
-        this,
-        _1);
-    ledger_->SavePendingContribution(std::move(non_verified), save_callback);
+  if (non_verified.empty()) {
+    return;
```

Comment:
No log needed here?

Reply:
not needed as if empty we don't need to do anything